test/e2e/config/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- ../../../config/default
+- ../../../config/dev
```

Comment:
This manager's deployment doesn't have settings to mount the manager's configuration.

So, should we sync [this](https://github.com/kubernetes-sigs/kueue/blob/452b92713eeb6cd30bfa765afc6340134966e65a/config/dev/manager_config_patch.yaml) with [default](https://github.com/kubernetes-sigs/kueue/blob/452b92713eeb6cd30bfa765afc6340134966e65a/config/default/manager_config_patch.yaml#L14-L21)?

Reply:
Not necessary.

`dev` uses `default` as its base. I got this in a live cluster: https://github.com/kubernetes-sigs/kueue/blob/452b92713eeb6cd30bfa765afc6340134966e65a/config/dev/kustomization.yaml#L5

```
- /manager
        image: gcr.io/k8s-staging-kueue/kueue:v0.6.0-devel-174-g77011c4-dirty
        imagePullPolicy: IfNotPresent
        livenessProbe:
          failureThreshold: 3
          httpGet:
            path: /healthz
            port: 8081
            scheme: HTTP
          initialDelaySeconds: 15
          periodSeconds: 20
          successThreshold: 1
          timeoutSeconds: 1
        name: manager
        ports:
        - containerPort: 8082
          name: visibility
          protocol: TCP
        - containerPort: 9443
          name: webhook-server
          protocol: TCP
        readinessProbe:
          failureThreshold: 3
          httpGet:
            path: /readyz
            port: 8081
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        resources:
          limits:
            cpu: 500m
            memory: 512Mi
          requests:
            cpu: 500m
            memory: 512Mi
        securityContext:
          allowPrivilegeEscalation: false
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
        volumeMounts:
        - mountPath: /tmp/k8s-webhook-server/serving-certs
          name: cert
          readOnly: true
        - mountPath: /controller_manager_config.yaml
          name: manager-config
          subPath: controller_manager_config.yaml
```

Reply:
Oh, it makes sense! Thank you for the clarifications :)
/lgtm
/approve